leshan-client-demo/src/main/java/org/eclipse/leshan/client/demo/LeshanClientDemo.java

```diff
@@ -191,31 +198,35 @@ public static LeshanClient createClient(LeshanClientDemoCLI cli, LwM2mModelRepos
                 initializer.setClassForObject(SERVER, Server.class);
             }
         } else {
+            int shortServerId = cli.main.factoryBootstrap == null || cli.main.factoryBootstrap.get("/1/0/0") == null
+                    ? 123
```

Comment:
I get the idea but I'm not totally comfortable with the idea to have a "special behavior" for resource "/1/0/0" 

To change shortServerId using `-fc` option, I expected that user will need to do : 
```
java -jar leshan-client-demo.jar -fc /1/0/0=1234,/0/0/10=1234
```
Does it sound OK to you ?

Reply:
This part of code is necessary. ShortServerId is setting earlier and if we delete this part of code the value won't be change using -fb option. User still need to do:
java -jar leshan-client-demo.jar -fb /1/0/0=1234
with this approach.
This problem occuours only with shortServerId.

Reply:
> This part of code is necessary. ShortServerId is setting earlier and if we delete this part of code the value won't be change using -fb option. 

I'm almost sure I tested it and it worked for me. :thinking: 
I will retest it.

Reply:
After fixing bug it works for me too 😉